packages/rest/README.md

```diff
@@ -27,20 +27,17 @@ Here's a basic "Hello World" application using `@loopback/core` and
 
   ```ts
   import {Application} from '@loopback/core';
```

Comment:
Unused import

Reply:
I don't think we need to install `@loopback/core` as a dependency anymore

Reply:
This might be one of those cases where the import is required for the compiler to make sense of another required type's dependencies. The easiest way would be to attempt to compile it without this line and find out!

Reply:
It's definitely an unused import. I've used `RestApplication` without installing / using `@loopback/core`.

Reply:
Interesting. I copied this example into a standalone file and compiled it using the following command:

```
tsc x.ts --target es2017 -m CommonJS
```

However, I received a bunch of errors about missing type information for Node.js. To fix that, I had to add `@types/node` to regular dependencies of `@loopback/rest`. With that change in place, the example compiles fine.